### PR TITLE
Create config.py

### PR DESCRIPTION
**Pull request by sils**
# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -2,16 +2,16 @@
 import os
 
 COBOT_ROOT = os.environ.get('COBOT_ROOT', os.getcwd())
-
+
```
Comment (nkprince007) [inline config.py]: Testing

**Comment by meetmangukiya** [comment]
ack 		eb07e35

**Comment by meetmangukiya** [comment]
unack 		eb07e35

**Comment by meetmangukiya** [comment]
I assume this test PR is of no use now, closing

**Pull request closed without merging**